### DISCUSSION TITLE
zone2sql records on powerdns base not show in interface powerdns-admin after migrate zones from bind

### DISCUSSION THREAD

**Discussion by urobasa** [ngoduykhanh/PowerDNS-Admin#1278] [General]
How to see in powerdns-admin zones that were transferred to the powerdns database using the zone2sql tool from bind.
export from powerdns database to powerdns-admin database? instructions?

**Comment by metrax** [comment]
If you have set bg_domain_updates to ON, than you have to refresh the list in PDA, by clicking the refresh button at the dasboards zone list or wait until the cron was executed.

If bg updates is of, the zone should be recordnized automatically.
